### PR TITLE
Customize homepage

### PR DESCRIPTION
**Pull request by ChristinaButtercup**
Added cards to Home Page
Make the button "Discover" work --> leads to "tours"
Changed the picture in the navbar into a logo
When clicking on the logo --> leads to "tours"

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <div class="container-fluid">
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= link_to tours_path, class: "navbar-brand" do %>
```
Comment (mariorzgz) [inline app/views/shared/_navbar.html.erb]: Can you make it go to the home page instead of the index of tours?

**Review by mariorzgz — COMMENTED** [review]
Don't you think it would be better if the logo goes to the homepage instead of the tours index?

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (ChristinaButtercup) [reply app/views/shared/_navbar.html.erb]: You are right, I will change

Comment (ChristinaButtercup) [reply app/views/shared/_navbar.html.erb]: I have changed it

**Pull request merged**